de.fu_berlin.inf.dpp.ui.frontend/html/src/components/MainView/ActiveAccount.jsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from 'react-localize'
+import React from 'react'
+
+class ActiveAccount extends React.Component {
+  render () {
+    const { activeAccount } = this.props
+    var accountText
+    var className
+
+    if (typeof activeAccount === 'object' && activeAccount.username && activeAccount.domain) {
```

Comment:
can the active Account be anything other than an object?

if you simply want to check that it is not null or not undefined you try this:
`if (activeAccount && activeAccount.username && activeAccount.domain)`

Reply:
Done